tags/logrus/server_interceptors.go

```diff
@@ -0,0 +1,32 @@
+package ctxlogger_logrus
+
+import (
+	"github.com/grpc-ecosystem/go-grpc-middleware"
+	"github.com/sirupsen/logrus"
+	"golang.org/x/net/context"
+	"google.golang.org/grpc"
+)
+
+// PayloadUnaryServerInterceptor returns a new unary server interceptors that adds logrus.Entry to the context.
```

Comment:
This doc line should be updated.

Reply:
Will fix or delete depending on the below 👍